verticapy/_utils/_sql/_format.py

```diff
@@ -695,15 +691,13 @@ def format_query(
             .replace("\n", "<br>")
             .replace("    ", "&nbsp;&nbsp;&nbsp;&nbsp;")
         )
-    if display_success:
```

Comment:
You are changing this print. Is that intentional? 

Even below where you change to 'elif print_sql'. You are adding the RETURN to print_sql. Just double check if that was intentional.